### ISSUE TITLE
Evaluation view  - loading students by pages

### ISSUE THREAD

**Issue by avukkonen** [otavanopisto/muikku#922]
Teacher Tuija Salminen reports:

>Arviointinäkymässä on kaikki M2-opiskelijatkin ja näin kuuluu ollakin, mutta minä en pääse katsoamaan monikulttuuristen opiskelijoiden tekemisiä, koska kolmannen välilehden jälkeen arviointinäkymä pyörittää vain rinkulaa. 10 minuuttia on odotellut, ei auta.

>sitkeästi kun yritin, sain pikkuhiljaa sivuja auki, minun piti nimittäin päästä sivuille neljö, viisi, kuusi, seitsemän jne. Näyttäisi siltä, että kun sivut saa kerran aukeamaan, niin sitten ne aukeaa jatkossa paremmin. 20 sek per sivu on huomattavasti parempi kuin 10 minuuttia per sivu ;)

**Comment by issakainen** [comment]
Now that evaluation tool has been revamped, is this still valid?

**Comment by jjlankinen** [comment]
Now everything should be much more usable and faster so moving this to done.
